jni/src/test/java/org/infinispan/client/jni/hotrod/JniTest.java

```diff
@@ -158,18 +155,17 @@ when main() returns. */
 
 @Override
 public boolean includeMethod(IMethodSelectorContext context, ITestNGMethod method, boolean isTestMethod) {
```

Comment:
It looks like the indentation for `includeMethod` and `setTestMethods` is off?

Reply:
ok, fixed